WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -267,6 +270,7 @@ public void showMagicLinkSentScreen(String email) {
     @Override
     public void openEmailClient() {
         if (WPActivityUtils.isEmailClientAvailable(this)) {
+            AnalyticsTracker.track(AnalyticsTracker.Stat.LOGIN_MAGIC_LINK_OPEN_EMAIL_CLIENT_VIEWED);
```

Comment:
Hey @aerych, I see, I will make the change.

We should probably rename this one to something more clear (like `LOGIN_MAGIC_LINK_SENT_VIEWED`) while it's still early.

We should probably also track the attempt to open the email client. May I suggest introducing a `LOGIN_MAGIC_LINK_OPEN_EMAIL_CLIENT_CLICKED`?